scripts/create-docs.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+TEMP_DIR=$(mktemp -d)
+
+cat keywords.robot > "$TEMP_DIR/all-keywords.robot"
+find lib -name "*.robot" -exec cat {} + >> "$TEMP_DIR/all-keywords.robot"
+
+python3 robot-venv/bin/libdoc "$TEMP_DIR/all-keywords.robot" "$TEMP_DIR/all-keywords.html" >/dev/null 2>&1
+
+if [ $? -ne 0 ]; then
+  echo "libdoc command failed"
+  exit 1
+fi
+
+cp "$TEMP_DIR/all-keywords.html" .
```

Comment:
@WiktorG351 where is `docs` directory used? yml has it specified

Reply:
@artur-rs adressed [here](https://github.com/Dasharo/open-source-firmware-validation/pull/352/commits/a2bc617ebb04f86174ec5ae2e46003f1aebd3a4f#diff-aa664a4f27e008df9a11dea5effa5d1adfa409490475f026c53ca44419606b81L15-R15)